blueprint/src/chapters/gaussian.tex

```diff
@@ -207,45 +209,46 @@ \subsection{Gaussian measures on a finite dimensional Hilbert space}
 \end{align*}
 \end{proof}
 
-Let $E$ be a finite dimensional Hilbert space. We denote by $\langle \cdot, \cdot \rangle$ the inner product on $E$ and by $\Vert \cdot \Vert$ the associated norm.
-
+Let $E$ be a separable Hilbert space. We denote by $\langle \cdot, \cdot \rangle$ the inner product on $E$ and by $\Vert \cdot \Vert$ the associated norm.
 
 \begin{lemma}\label{lem:IsGaussian.charFun_eq}
-  \uses{def:IsGaussian, def:charFun, def:covMatrix}
+  \uses{def:IsGaussian, def:charFun, def:covInnerBilin}
+  \leanok
+  \lean{ProbabilityTheory.IsGaussian.charFun_eq}
 The characteristic function of a Gaussian measure $\mu$ on $E$ is given by
 \begin{align*}
-  \hat{\mu}(t) = \exp\left(i \langle t, m_\mu \rangle - \frac{1}{2} \langle t, \Sigma_\mu t \rangle\right) \: .
+  \hat{\mu}(t) = \exp\left(i \langle t, m_\mu \rangle - \frac{1}{2} C(t, t)\right) \: .
```

Comment:
The covariance bilinear form for \mu in a Hilbert space is denoted by `C'_\mu` (`C_\mu` is the one defined on the dual). Putting a prime on the most used one was possibly not a great idea, but let's keep is consistent for now.
```suggestion
  \hat{\mu}(t) = \exp\left(i \langle t, m_\mu \rangle - \frac{1}{2} C'_\mu(t, t)\right) \: .
```